invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/button.overrides

```diff
@@ -56,7 +56,7 @@
 }
 
 .ui.button.fluid-mobile {
-  @media all and (max-width: @largestMobileScreen) {
+  @media all and (max-width: @largestTabletScreen) {
```

Comment:
Can we rather add another class for tablet `.fluid-tablet` with `@media screen and (min-width: @tabletBreakpoint) and (max-width: @largestTabletScreen)` , so that it's not affecting other cases where the button should not be fluid on tablet? Then apply both `.fluid-mobile` and `.fluid-tablet` to the button that should be fluid on both screen sizes?